### PR TITLE
Redesign proposal navigation

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Update proposal navigation style.

# Changes

- move title into navigation block
- add universe logo
- buttons style

# Tests

Updated.

# Todos

- [ ] Add entry to changelog (if necessary).
Not necessary

# Screenshots

## Flag enabled

image
image
image

## Flag disabled
image

image

### CODE REVIEW

File: frontend/src/tests/lib/components/proposal-detail/ProposalNavigation.spec.ts [hunk 1]
```diff
@@ -9,25 +11,43 @@ describe("ProposalNavigation", () => {
     return ProposalNavigationPo.under(new JestPageObjectElement(container));
   };
 
+  beforeEach(() => {
+    // for logo
+    page.mock({ data: { universe: OWN_CANISTER_ID_TEXT } });
```
Comment (lmuntaner) [inline frontend/src/tests/lib/components/proposal-detail/ProposalNavigation.spec.ts:16]: How about one describe with the feature flag disabled and the same old tests.

Another describe with the feature flat enabled and these tests?

**Review by lmuntaner — COMMENTED** [review]
A few comments, thanks!

File: frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte [hunk 2]
```diff
@@ -29,10 +38,54 @@
     assertNonNullish(olderId);
     selectProposal(olderId);
   };
+
+  let title: string;
+  $: title = `${$i18n.proposal_detail.title} ${currentProposalId}`;
```
[the remark this answers is not in the document]
Comment (mstrasinskis) [reply frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte:43]: Good point, but the same approach as for [the page title](https://github.com/dfinity/nns-dapp/blob/27ad98d0b4f2dc1319459900b47578e934d2f12c/frontend/src/lib/pages/NnsProposalDetail.svelte#L124) was used. I think it's better to use same label for same text. Let me know if you're not convinced :)

File: frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte [hunk 3]
```diff
@@ -29,10 +38,54 @@
...
-  <div role="toolbar" data-tid="proposal-nav">
+{#if $ENABLE_FULL_WIDTH_PROPOSAL}
+  <div class="proposal-nav" role="toolbar" data-tid="proposal-nav">
+    <span class="status">
+      TBD: Status
+      <!-- TODO(GIX-1971): render proposal status component -->
+    </span>
+    <h2 class="title">
+      <div class="universe-logo">
+        <UniverseLogo
```
[the remark this answers is not in the document]
Comment (mstrasinskis) [reply frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte:54]: Here only logo(icon) should be displayed.

File: frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte [hunk 4]
```diff
@@ -29,10 +38,54 @@
...
+        />
+      </div>
+      <TestIdWrapper testId="title">{title}</TestIdWrapper>
+    </h2>
+    <button
+      class="ghost newer"
+      type="button"
+      aria-label={$i18n.proposal_detail.newer}
+      on:click={selectNewer}
+      class:hidden={isNullish(newerId)}
```
[the remark this answers is not in the document]
Comment (mstrasinskis) [reply frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte:68]: To preserve the space.

File: frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte (hunk 2 above)
[the remark this answers is not in the document]
Comment (lmuntaner) [reply frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte:43]: Ah, ok, maybe you can change it in another PR then?

File: frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte (hunk 3 above)
[the remark this answers is not in the document]
Comment (lmuntaner) [reply frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte:54]: Oops, sorry, yes, good point

File: frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte [hunk 5]
```diff
@@ -29,10 +38,54 @@
...
+      data-tid="proposal-nav-newer"
+    >
+      <IconLeft />
+      {$i18n.proposal_detail.newer_short}</button
+    >
+    <button
+      class="ghost older"
+      type="button"
```
[the remark this answers is not in the document]
Comment (mstrasinskis) [reply frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte:76]: Sorry, could you please describe the question more precisely?

File: frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte (hunk 2 above)
[the remark this answers is not in the document]
Comment (mstrasinskis) [reply frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte:43]: [Created](https://dfinity.atlassian.net/browse/GIX-1993) 👍

File: frontend/src/tests/lib/components/proposal-detail/ProposalNavigation.spec.ts (hunk 1 above)
Comment (mstrasinskis) [reply frontend/src/tests/lib/components/proposal-detail/ProposalNavigation.spec.ts:18]: Makes sense. Added.

**Review by lmuntaner — APPROVED** [review]
Thanks!

File: frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte (hunk 5 above)
[the remark this answers is not in the document]
Comment (mstrasinskis) [reply frontend/src/lib/components/proposal-detail/ProposalNavigation.svelte:76]: This is copy/paste from the current version (+new icons) to reuse in another layout.

**Pull request merged**